src/panels/ShareModelPanel/ShareModel.tsx

```diff
@@ -149,15 +149,11 @@ export default function ShareModel() {
       );
     } catch (error) {
       reactHotToast.custom((t) => (
-        <ToastCard
-          toastInstance={t}
-          type="negative"
-          text={
-            typeof error === "string"
-              ? error
-              : "Unable to update model permissions"
-          }
-        />
+        <ToastCard toastInstance={t} type="negative">
+          {typeof error === "string"
+            ? error
```

Comment:
Don't think this is of high importance, but maybe we could test this code path?

Reply:
Done.